src/test_utils.rs

```diff
@@ -310,7 +314,11 @@ impl TestContext {
     /// Gets the most recent message over all chats.
     pub async fn get_last_msg(&self) -> Message {
         let chats = Chatlist::try_load(&self.ctx, 0, None, None).await.unwrap();
-        let msg_id = chats.get_msg_id(chats.len() - 1).unwrap();
+        let msg_id = chats.get_msg_id(0).unwrap();
+        // 0 is correct here (as opposed to `chats.len() - 1`, which would be the last element):
```

Comment:
good catch! maybe move the comment above `let msg_id = chats.get_msg_id(0).unwrap();` (imu, comments usually described what follows)